src/platform/native/Core.zig

```diff
@@ -684,6 +684,7 @@ pub fn update(self: *Core, app: anytype) !bool {
 
     if (self.window.shouldClose()) {
         self.pushEvent(.close);
+        self.window.setShouldClose(false);
```

Comment:
This line should perhaps be above the `pushEvent`?

Reply:
Sure thing!